packages/core-modules/contracts/submodules/election/ElectionSchedule.sol

```diff
@@ -17,6 +17,10 @@ contract ElectionSchedule is ElectionBase {
     }
 
     function _getCurrentPeriodType() internal view returns (ElectionPeriod) {
```

Comment:
You can use the onlyIfInitialized modifier

Reply:
Yep, this is something I considered and chose to go with this design, so I wouldn't have to put `onlyIfInitialized` all over the place.

But I do see your point. This may be a form of semantic overloading. If you consider it to be so, would you mind merging this PR and letting me know, so that I create a new issue to remove the overload? It would simplify my work on the evaluations feature.

Reply:
Sure, creating a ticket for that.